base/deprecated.jl

```diff
@@ -259,3 +259,5 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool()      rand(Bool)
+@deprecate randbool(x...)  bitrand(x)
```

Comment:
@rfourquet Thanks. I have added all the deprecation cases, and they work correctly now.